resources/assets/components/Tags/index.js

```diff
@@ -76,11 +75,14 @@ class Tags extends React.Component {
       'group-photo': 'Group Photo',
     };
 
-    const showTags = this.props.disabled ? 'disabled' : 'enabled';
-
     return (
       <div>
-        <h4 className={showTags}>Tags</h4>
+        <h4>
+          Tags (see{' '}
+          <a href="/faq#tags" target="_blank">
```

Comment:
I opted to open this one in a new tab since otherwise, clicking the link would cause you to lose the post you'd just reviewed before being able to add tags.